bitrepository-integrity-service/src/main/java/org/bitrepository/integrityservice/cache/PillarCollectionStat.java

```diff
@@ -21,6 +21,12 @@
  */
 package org.bitrepository.integrityservice.cache;
 
+import org.bitrepository.common.utils.TimeUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Intellij IDEA inserted this import automatically. I suppose we don’t want the dependency?

Reply:
I don't really know how to feel about this import.. I don't see a problem in removing it here, but IntelliJ annoyingly complains if the one in `DefaultThreadFactory` is removed (some explanation here: https://stackoverflow.com/questions/64156483/not-annotated-parameter-overrides-notnull-parameter)

Reply:
If we don’t intend on getting completely rid of the dependency, I suggest it’s fine to use it here too.